learnEcl/1200-dataset.md

```diff
@@ -7,19 +7,19 @@ slug: dataset
 
 A Dataset is a set of records that an ECL program can manipulate. A Dataset can be initialized using a logical file (File Dataset), inline data (Inline Dataset) or by another ECL function that filters, queries or transforms data (Derived Dataset).
 
-When defining DATASET you need to define the [RECORD](/learn-ecl/record.md) of the dataset first.
+When defining a DATASET you need to define the [RECORD](./1100-record.md) of the dataset first.
 
 ## SQL vs. ECL
 
 DATASET is similar to TABLE in SQL.
 
-| SQL                                                                                               | ECL                                                        |
-| ------------------------------------------------------------------------------------------------- | ---------------------------------------------------------- |
+| SQL | ECL |
+| - | - |
 | SimpleTable = select \* from '~simpledata.csv' type csv layout SimpleLayout as simpleDS offset 1; | simpleDS := DATASET('~simpledata.csv', SimpleLayout, CSV); |
 
 ## Inline Dataset
 
-A temporary dataset that's created and used while job is running. Inline dataset definition can be used for small datasets.
+A temporary dataset that's created and used while a job is running. Inline dataset definition can be used for small datasets.
```

Comment:
Inline datasets are static (and usually small) datasets defined in the code, not temporary datasets.